jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudformation/CfnProblem.kt

```diff
@@ -0,0 +1,8 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.cloudformation
+
+import com.intellij.psi.PsiElement
+
+class CfnProblem(val element: PsiElement, val description: String)
```

Comment:
It is always the red errors. I do not know if there are any warnings generated in the original plugin

I do know it will be re-used when we validate against resources. Given that, want to stick to CfnValidationError?

Reply:
Ah if it's used there then makes sense as is